.eslintignore

```diff
@@ -27,4 +38,9 @@ src/components/TagColorsModal.js
 src/components/TextBadge.js
 src/components/WithAutocard.js
 src/components/WithLoading.js
+
+.git
+dist
+node_modules
```

Comment:
It's needed for `esprint` which I was playing around with. Can remove it from this pr though.

Reply:
Unless there's some tangible benefit to `esprint`, I'd rather just remove it altogether.

Reply:
esprint makes it much faster to run eslint which was useful for detecting introduced errors in refactoring since it let me run it on every file save (only including the dependent files).